db/background_mgr.go

```diff
@@ -63,42 +97,163 @@ func (b *BackgroundManager) Start(options map[string]interface{}) error {
 		return err
 	}
 
+	var processClusterStatus []byte
+	if b.clusterAwareOptions != nil {
+		processClusterStatus, _, err = b.clusterAwareOptions.bucket.GetRaw(b.clusterAwareOptions.StatusDocID())
+		if err != nil && !base.IsDocNotFoundError(err) {
+			return pkgerrors.Wrap(err, "Failed to get current process status")
+		}
+	}
+
 	b.resetStatus()
+
+	err = b.Process.Init(processClusterStatus)
+	if err != nil {
+		return err
+	}
+
+	if b.clusterAwareOptions != nil {
+		go func() {
+			ticker := time.NewTicker(BackgroundManagerStatusUpdateIntervalSecs * time.Second)
+			for {
+				select {
+				case <-ticker.C:
+					// Implement a retry on here. It's not the end of the world if we miss a couple of these, but we
```

Comment:
We now have retry down in UpdateStatucClusterAware - can we remove this comment?